lerna.json

```diff
@@ -2,7 +2,7 @@
   "lerna": "2.9.0",
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "0.2.11",
+  "version": "0.3.0-prerelease",
```

Comment:
Is this intentional?

Reply:
yes, because it ensures that the next npm release would be 0.3.0.

As this PR contains breaking changes, we want to ensure the next version after merge is 0.3.0.

Reply:
you can test this in a new folder:

```
mkdir test-prerelease
cd test-prerelease
npm init 0.3.0-prerelease
cat package.json
npm version patch
cat package.json
```